Taggers/python/flashggDoubleHReweight_cfi.py

```diff
@@ -1,8 +1,22 @@
 import FWCore.ParameterSet.Config as cms
 
+#default values first
+weightsFile = "flashgg/Taggers/data/HHTagger/HHreweight_2016nodes_04022018.root"
```

Comment:
Is this a joke? Why don't you just use the metaConditions?

Reply:
same later on ...

Reply:
My mistake, I forgot to put default values for reweighting to the MetaConditions. Fixed now. I also set all default values in the config files to empty. In this case if MetaConditions are not properly filled, it will crash.